doc/doxygen/install/install-linux.doxygen

```diff
@@ -109,6 +109,7 @@
     sudo add-apt-repository universe
     sudo apt update
     sudo apt-get install build-essential cmake autoconf patch libtool git automake
+    # note: QtSVG might also be available using 'qt6-svg-dev'
```

Comment:
Thanks. Doing a quick research I would actually recommend having qt6-svg-dev as main suggestion. The libqt6svg6-dev only exists on sid unstable

Reply:
ok, done